BrightID/src/components/Helpers/NodeModal.tsx

```diff
@@ -66,6 +80,100 @@ const NodeModal = () => {
     );
   }
 
+  const [httpServerUrl, setHttpServerUrl] = useState('');
+
+  const password = useSelector((state) => state.user.password);
+
+  const startHttpServer = useCallback(async () => {
+    httpBridge.start(
```

Comment:
Please move the logic to run the http server to a separate file/module and just trigger start/stop from here.

Reply:
Done